cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -281,28 +281,6 @@ arrow::Result<std::shared_ptr<arrow::RecordBatch>> makeUncompressedRecordBatch(
   return arrow::RecordBatch::Make(writeSchema, 1, {arrays});
 }
 
-class EvictGuard {
- public:
-  explicit EvictGuard(SplitState& splitState) : splitState_(splitState) {
-    oldState_ = splitState;
-    splitState_ = SplitState::kUnevictable;
```

Comment:
The definition for `kUnevictable` should be removed too.

Reply:
We cannot remove it. This state prevents re-entry into the eviction code block.